src/functions/TestResults.ps1

```diff
@@ -116,7 +116,7 @@ function Export-JUnitReport {
     -Passthru or by using the Run.PassThru configuration-option.
 
     .PARAMETER Path
-    The path where the XML-report should  to the ou the XML report as string.
+    The path where the XML-report should output the XML report as string.
```

Comment:
```suggestion
    The path where the XML-report should be saved.
```

See previous comment

Reply:
great suggestion!  "The path where the XML-report should be saved.